src/components/showtimesList.js

```diff
@@ -1,14 +1,15 @@
 import ShowtimesListDay from "./showtimesListDay";
 import styles from "@/styles/asideShowTimes.module.scss";
 
-const showtimesList = ({ showtimes }) => {
+const showtimesList = ({ showtimes, movies }) => {
```

Comment:
There's some prop drilling here which I will change if I have time.. for now, this works!